### PR TITLE
Add StatsD Profiling To Authentication

### PR DESCRIPTION
**Pull request by mloberg**
I added some StatsD collection around new users and the total time for a login/logout request.

### CODE REVIEW

**Comment by mloberg** [comment]
TODO: Add tags for LOA 1 vs LOA 3

**Comment by saneshark** [comment]
I don't have any thoughts on benchmarking, but I would agree having a common set of patterns we use for benchmarking would be nice so that we have some consistency in understanding results on grafana, etc. Maybe one way to think about it would be: How would we use the benchmark to trigger alerts or pager duty and work backward from there?

File: spec/lib/benchmark/timer_spec.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
     Benchmark::Timer.start('test', 'foo')
 
     expect(redis.get('benchmark_test_foo')).to_not be_nil
-    expect(redis.ttl('benchmark_test_foo')).to be > 0
+    expect(redis.ttl('benchmark_test_foo').positive?).to be true
```
Comment (jkassemi) [inline spec/lib/benchmark/timer_spec.rb]: weird - did the first version not do what was expected?

Comment (jkassemi) [reply spec/lib/benchmark/timer_spec.rb]: n/m, just saw the slack discussion around rubocop.

**Comment by mloberg** [comment]
I need to update the tags for StatsD so they get into Prometheus as well. It needs to be in the format of `name:value`.

**Comment by mloberg** [comment]
Closing in favor of #1399 for current state. Will work with identity team for changes related to identity flows

**Pull request closed without merging**